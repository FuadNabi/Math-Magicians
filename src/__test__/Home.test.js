import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('Home page test', () => {
  const tree = renderer.create(<Home />).toJSON;
  expect(tree).toMatchSnapshot();
});
