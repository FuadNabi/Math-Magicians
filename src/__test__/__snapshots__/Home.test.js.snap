// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page test 1`] = `[Function]`;
